docs/help/specifications-support/openapi-support/name-and-sort-resources.md

```diff
@@ -44,54 +54,68 @@ There are three operations here, and three different ways to generate, group and
 
 ![Group by path or group by tag?](/files/help/legacy/-MU8HgDhZXrUBOih-Aa4.png)
 
-### Group by path
-
-In this case, we deduce endpoint name from related `path`. First part of the path is extracted to generate the `endpoint` name, and every operation related to this endpoint name will be grouped together.
-
-
-Let's Bump it up to something visual with a first [Petstore live example](https://bump.sh/hub/examples/doc/petstore):
-
-<div style={{textAlign: 'center'}}>
-
-![Group petstore operations by path](/files/help/group-by-path-dark.png)
-
-</div>
+### Group by tag
 
-Operations `GET`  and `POST` from `/pets` , and `GET` from `/pets/{petId}` are grouped under same endpoint `Pets` (first part of the path), and their names are generated from field `summary`
+:::caution
+Be careful about operations without tags when option `Group operations by tag` is selected, as they would be ignored!
+:::
 
-### Group by tag
+You may have noticed in the above API Contract that there are two different `tags`: `all pets` and `single pet`.
+If you choose option `Group by tag`, Bump.sh will use these `tags` to group, name and sort the operations.
 
-You may have noticed they are two different `tags` for these 3 operations: "all pets" and "single pet" . If you chose option "Group by tag", we'll use these values to group, name and sort these operations. Here is our second [Petstore live example](https://bump.sh/hub/examples/doc/petstore-grouped-by-tags), based on the same specification, with option "Group by tag" :
+ Here is our second [Petstore live example](https://bump.sh/hub/examples/doc/petstore-grouped-by-tags), based on the same specification, with option `Group by tag` :
```

Comment:
You mention a “second” example here, and the “first” later in the `### Group by path` paragraph, should the numbering be reversed, or should the paragraphs be reversed?

Reply:
Fixed in https://github.com/bump-sh/docs/pull/68/commits/7ad750470fd89ac38bc6f45a6194acb5438a64db